Libplanet/Blockchain/Policies/VolatileStagePolicy.cs

```diff
@@ -0,0 +1,152 @@
+#nullable enable
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Threading;
+using Libplanet.Action;
+using Libplanet.Tx;
+
+namespace Libplanet.Blockchain.Policies
+{
+    /// <summary>
+    /// In-memory staged transactions.
+    /// </summary>
+    /// <typeparam name="T">An <see cref="IAction"/> type.  It should match
+    /// to <see cref="BlockChain{T}"/>'s type parameter.</typeparam>
+    public class VolatileStagePolicy<T> : IStagePolicy<T>
+        where T : IAction, new()
+    {
+        private readonly ConcurrentDictionary<TxId, Transaction<T>> _set;
+
+        private readonly List<TxId> _queue;
+
+        private readonly ReaderWriterLockSlim _lock;
+
+        /// <summary>
+        /// Creates a new <see cref="VolatileStagePolicy{T}"/> instance.
+        /// </summary>
+        public VolatileStagePolicy()
+        {
+            _set = new ConcurrentDictionary<TxId, Transaction<T>>();
```

Comment:
It seems there is no code to delete `_set`'s element. Is there a default capacity?

Reply:
No, we need to implement lifetiming, but I'm going to do it on a different PR.